lcd/mini-12864/btt-octopus-11.cfg

```diff
@@ -0,0 +1,11 @@
+[display]
```

Comment:
I chose for separate files because the spider uses a different spi bus (which turns out to share its CS pin with the adxl so they cant be used at the same time). afaik we cannot use aliases for busses, If there is a cleaner way to do this I am all for it but for know this was the cleanest and easiest I could think of.

Reply:
Aaah i see what you mean. I think that's an issue that should be resolved with the spider wiring though, i saw they were still moving these around because someone borked a mini 12864. I'll figure out where we are with that. Preferably we shouldn't have pin intersections like that. Put a pin in that for now, i'll dig deeper.

Reply:
Hi! I'm chiming in here because I saw your discussion about the SPI bus on the spider. Indeed there was a little bit of discussion about that.
We came to the conclusion that the best SPI to use is the one on that external SD-Card header, which is also shared with the display and the interal SD-Card. As long as no card is inserted, the ADXL works fine and it doesn't mess with the display afaik.
That one borked mini 12864 was just someone with a v1.0 Spider that has swapped EXP1/EXP2 labels - he swapped his connections on the board, then everything worked again.